prowler/lib/persistence/manager.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from .sqlite import SQLiteList, SQLiteDict
+
+default_list = SQLiteList
+
+
+def mklist() -> list:
+    """
+    Create a new list with the given name and data.
+    """
+    prowler_db_connection = os.environ.get('PROWLER_DB_CONNECTION', "memory://")
```

Comment:
I'm confused about the usage of this environment variable. Why does it have to be a string? Wouldn't it make more sense to be a flag to either use sqlite or not?

Reply:
I can only see this variable usage as a flag, unless I'm missing something. That would make its name not descriptive enough.

Reply:
For now, we will leave it as is, looking to have a DB connection string within the environment variable.